index.js

```diff
@@ -41,6 +41,14 @@ let supportLevel = (() => {
 		return 1;
 	}
 
+	if ('CI' in env) {
+		if (['TRAVIS', 'CIRCLECI', 'APPVEYOR', 'GITLAB_CI'].some(sign => sign in env)) {
+			return 1;
+		}
+
+		return 0;
```

Comment:
@not-an-aardvark 
Can you try to delete this line for build for eslint?

Reply:
I tried deleting that line and then running `CI=1 TRAVIS=1 npm test` on the ESLint repo (to simulate a build on Travis). However, it didn't seem to fix the issue.

It seems like the behavior change is occurring because `supports-color` is now returning `1` on Travis in a case where it previously returned `0`. So I think removing a `return 0` line is unlikely to fix the issue, because that line would only get reached when `supports-color` is correctly returning 0.